pages/apis/managing_api_tokens.md

```diff
@@ -60,6 +60,14 @@ Removing access from a token will send a notification email to the token's owner
 >📘
 > Removing access from a token does not delete the token, however token owner cannot re-add your organization to their token's scope.
 
+## Limiting API Access by IP address
+>🚧 Beta Feature
+> This feature is not yet available to all organizations.
+
+If you'd like to limit access to your organization by IP address, you can create an allow list of IP addresses on the [organization's member permissions page](https://buildkite.com/organizations/~/member-permissions).
+
+The allow list can also be managed using the [GraphQL API](/docs/apis/graphql-api) with the `organizationAPIIPAllowListUpdate` mutation.
```

Comment:
Nitpicks:

- allow list -> allowlist
- Passive voice -> active
- Link to the specific mutation (I assume that's the one you were waiting for)

```suggestion
You can also manage the allowlist with the [`organizationAPIIPAllowListUpdate`](/docs/apis/graphql/schemas/mutation/organizationapiipallowlistupdate) mutation in the GraphQL API.
```

Reply:
Is it too late to rename the mutation? I think it should be `organizationApiIpAllowlistUpdate `

Reply:
Nope! PR here: https://github.com/buildkite/buildkite/pull/11543

Reply:
Above PR merged. Updated to `organizationApiIpAllowlistUpdate` here.